### PR TITLE
Update docker stack

### PR DESCRIPTION
**Pull request by dalg24**
pushed the dockerfile corresponding to the new image I pushed to docker hub last night

### CODE REVIEW

File: docker/Dockerfile_stack [hunk 1]
```diff
@@ -209,22 +211,39 @@ RUN export TRILINOS_VERSION=12.4.2 && \
...
+        -D Trilinos_ENABLE_Belos:BOOL=ON \
+        -D Trilinos_ENABLE_Epetra:BOOL=ON \
+        -D Trilinos_ENABLE_EpetraExt:BOOL=ON \
+        -D Trilinos_ENABLE_Ifpack:BOOL=ON \
+        -D Trilinos_ENABLE_Intrepid:BOOL=ON \
+        -D Trilinos_ENABLE_Kokkos:BOOL=ON \
         -D Trilinos_ENABLE_ML:BOOL=ON \
-        -D Trilinos_ENABLE_MueLu:BOOL=ON \
-        -D Trilinos_ENABLE_Tpetra:BOOL=ON \
+        -D Trilinos_ENABLE_MueLu:BOOL=OFF \
```
Comment (Rombur) [inline docker/Dockerfile_stack:224]: Can you compile deal.II with MueLu off?

**Comment by dalg24** [comment]
Yes, release 8.3.0 at least.

**Comment by dalg24** [comment]
I don't think it's going to crash configure 

https://github.com/dealii/dealii/blob/master/cmake/configure/configure_2_trilinos.cmake#L65

**Comment by dalg24** [comment]
never mind...

https://github.com/dealii/dealii/blob/master/cmake/configure/configure_2_trilinos.cmake#L71

**Comment by Rombur** [comment]
I know, I am the one who added MueLu to deal.II. That's why I am surprised that it works.

**Pull request merged**